src/main/java/seedu/address/logic/commands/order/AddCommand.java

```diff
@@ -25,7 +25,7 @@
             + PREFIX_PHONE + "PHONE "
             + PREFIX_ADDRESS + "ADDRESS "
             + PREFIX_FOOD + "FOOD...\n"
-            + "Example: " + COMMAND_WORD + " "
+            + "Example: " + OrderCommand.COMMAND_WORD + " " + COMMAND_WORD + " "
```

Comment:
Do add the Date in the example, which was merged in recently

Reply:
Update the javadocs for this class `person` -> `order`, `address book` -> `order book`